Documentation/Contributors/CodingGuide/README.md

```diff
@@ -182,6 +182,7 @@ Cartesian3.fromDegrees = function (
 
 ```javascript
 "use strict";
+
```

Comment:
This one is a little weird, but this README is actually out of date as well (since we use ES6 modules "use strict" is unneeded) so I'm not going to worry about it.